web/web.go

```diff
@@ -541,19 +541,39 @@ func (h *Handler) consoles(w http.ResponseWriter, r *http.Request) {
 	for k, v := range rawParams {
 		params[k] = v[0]
 	}
+
+	externalLabels := map[string]string{}
+	h.mtx.RLock()
+	els := h.config.GlobalConfig.ExternalLabels
+	h.mtx.RUnlock()
+	for _, el := range els {
```

Comment:
I think it would be wise to have a quick unittest either here or for the alerting rules to check that nil hasn't been passed to the templating.

Reply:
Yes, definitely. Will add at my next convenience.